jailer.go

```diff
@@ -318,6 +318,9 @@ func jail(ctx context.Context, m *Machine, cfg *Config) error {
 		WithSeccompLevel(cfg.JailerCfg.SeccompLevel).
 		WithStdout(stdout).
 		WithStderr(stderr).
+		WithStdout(os.Stdout).
+		WithStderr(os.Stderr).
```

Comment:
We're already calling WithStderr on line 320.

Reply:
Same here

Reply:
Can we add support for Stdin while we're at it?